docs/userGuide/syntax/thumbnails.mbdf

```diff
@@ -0,0 +1,46 @@
+## Thumbnails
+
+**A `thumbnail` component allows you to insert thumbnails using text, images, or icons.**
+
+<include src="outputBox.md" boilerplate >
+<span id="code">
+
+```html
+<thumbnail circle src="{{baseUrl}}/images/deer.jpg" size="150"/>
+<thumbnail circle text=":book:" background="#333" fontcolor="white" size="100"/>
+<thumbnail circle text="___CS___" background="#ccf" fontcolor="#339" border="3px solid #339" size="100"/>
+<thumbnail text=":fas-book:" fontcolor="darkgreen" size="100"/>
+```
+</span>
+<span id="output">
+<thumbnail circle src="{{baseUrl}}/images/deer.jpg" size="150"/>
+<thumbnail circle text=":book:" background="#333" fontcolor="white" size="100"/>
+<thumbnail circle text="___CS___" background="#ccf" fontcolor="#339" border="3px solid #339" size="100"/>
+<thumbnail text=":fas-book:" fontcolor="darkgreen" size="100"/>
+</span>
+</include>
+
+****Options****
+Name | Type | Default | Description
+--- | --- | --- | ---
+alt | `string` | | The browser displays this if the image fails to load
+border | `string` | | Accepts any valid CSS border property
+circle | `boolean` | false | If this option is turned on, the thumbnail will be cropped to a circle
+size | `string` | 100 | The size of the thumbnail in pixels
+src | `string` | | The URL of the image.<br>The URL can be specified as absolute or relative references. More info in: _[Intra-Site Links]({{baseUrl}}/userGuide/formattingContents.html#intraSiteLinks)_
```

Comment:
Likewise same comment for `src`.

Reply:
For thumbnail, `src` is not strictly required, since a `text` property may be specified.

We also support the case where both `src` and `text` are empty, by simply displaying an empty div of the specified size

Thus, neither of them are required attributes. Perhaps the documentation could be updated to reflect this more accurately